.github/workflows/test.yml

```diff
@@ -0,0 +1,148 @@
+# Check documentation at https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions
+name: Tests
+
+on:
+  pull_request: ~
+  schedule:
+    - cron: '0 2 * * *'
+  push:
+    branches:
+      - master
+
+jobs:
+  security-check:
+    name: Security Checker
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Setup PHP
+        uses: shivammathur/setup-php@v1
+        with:
+          php-version: 7.4
+          extensions: curl, dom, iconv, intl, json, libxml, mbstring, phar, tokenizer, xdebug, xml, xmlwriter
+
+      - name: Get Composer Cache Directory
+        id: composer-cache
+        working-directory: api
+        run: echo "::set-output name=dir::$(composer config cache-files-dir)"
+
+      - name: Cache dependencies
+        uses: actions/cache@v1
+        with:
+          path: ${{ steps.composer-cache.outputs.dir }}
+          key: ${{ runner.os }}-composer-${{ hashFiles('**/composer.json') }}
+          restore-keys: ${{ runner.os }}-composer-
+
+      - name: Install dependencies
+        working-directory: api
+        run: composer install --prefer-dist --no-progress --no-suggest --no-interaction
+
+      - name: Security check
+        working-directory: api
+        run: bin/console security:check
+
+  doctrine-schema-validator:
+    name: Doctrine Schema Validator
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Setup PHP
```

Comment:
Wouldn't it be better to use our own Docker images even here?

Reply:
No cause the `container.image` option on GitHub Actions does not support environment variables. So the full Docker image name will be public (`eu.gcr.io/{GKE_PROJECT}`)